README.md

```diff
@@ -249,7 +249,6 @@ and CITGM team members listed below.
 - [@BridgeAR](https://github.com/BridgeAR) - Ruben Bridgewater
 - [@cjihrig](https://github.com/cjihrig) - Colin Ihrig
 - [@codebytere](https://github.com/codebytere) - Shelley Vohr
-- [@evanlucas](https://github.com/evanlucas) - Evan Lucas
```

Comment:
He'll need to be removed from the GitHub [releasers group](https://github.com/orgs/nodejs/teams/releasers/members) otherwise he'll reappear the next time we sync the lists with `node-core-utils`.

Reply:
I am not a maintainer of the releasers group. @nodejs/releasers are you fine to promote me to maintainer so that I can update the group as well?

Reply:
I removed him from the releasers team